insolar/rootdomain/rootdomain.go

```diff
@@ -18,50 +18,28 @@ package rootdomain
 
 import (
```

Comment:
do we still need this package?

Reply:
I propose to make it another pull-quest. Too many changes already.if you agree I will create the task